src/test/java/com/truelayer/java/acceptance/PayoutsAcceptanceTests.java

```diff
@@ -69,4 +69,21 @@ public void shouldCreateAPayoutAndGetPayoutsDetails() {
         assertEquals(payoutId, getPayoutResponse.getData().getId());
         assertEquals(merchantAccount.getId(), getPayoutResponse.getData().getMerchantAccountId());
     }
+
+    public static Stream<Arguments> providePayoutsArgumentsForDifferentCurrencies() {
+        return Stream.of(
+                Arguments.of(
+                        CurrencyCode.GBP,
+                        AccountIdentifier.sortCodeAccountNumber()
+                                .accountNumber("00009650")
+                                .sortCode("040668")
+                                .build(),
+                        SchemeSelection.instantPreferred().build()),
+                Arguments.of(
+                        CurrencyCode.PLN,
+                        AccountIdentifier.iban().iban("GB25CLRB04066800046876").build(),
```

Comment:
We can't use the NRB here because the merchant account generated on the console has a GB IBAN, so NRB validation fails on MAP side (more info in [this](https://truelayer.slack.com/archives/C01JKPG129X/p1741255165944919) Slack thread)